.github/workflows/virtual-integration.yml

```diff
@@ -589,7 +589,19 @@ jobs:
           kubectl logs -n orch-app -l app=app-deployment-manager --tail=-1 > smoke-test/logs/app-deployment-manager.log
           kubectl logs -n orch-app -l app=app-resource-manager -c app-resource-manager --tail=-1 > smoke-test/logs/app-resource-manager.log
           kubectl logs -n orch-app -l app.kubernetes.io/name=app-orch-catalog --tail=-1 > smoke-test/logs/application-catalog.log
-          kubectl exec -n enic enic-0 -c edge-node -- bash -c "journalctl -xeu cluster-agent" > smoke-test/logs/enic-cluster-agent-post.log
+          echo "Collect logs from all pods in orch-infra namespace"
```

Comment:
We already do most of this work as part of the deploy-kind action, however the step to upload the log is not running. https://github.com/open-edge-platform/edge-manageability-framework/blob/main/.github/actions/deploy_kind/action.yaml#L113 

I think we should modify that step to something like below:

```
- name: Upload diagnostic information to CI artifact store
  if: always()
  uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02
  with:
    name: kind-diagnostics
    path: |
      kind-diagnostics/pods-list.txt
      kind-diagnostics/pods-describe.txt
      kind-diagnostics/argo-diag.txt
      kind-diagnostics/argocd-applications.yaml
      kind-diagnostics/events.yaml
    if-no-files-found: warn   # or 'ignore' for absolute robustness
```